assets/sass/06-components/entry.scss

```diff
@@ -215,6 +215,7 @@ body:not(.single) .site-main > article:last-of-type .entry-footer {
 		max-width: 100%;
 		margin-left: auto;
 		margin-right: auto;
+		margin-top: calc(2.0 * var(--global--spacing-vertical));
```

Comment:
Changing this to 2 instead of 2.0 should solve the style lint error

Reply:
Phew, thank you so much @carolinan